internal/fuzzbinder/fuzzclaim.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package fuzzbinder provides a function for generating a fuzzing claim
+// for a revision of a source code.
+package fuzzbinder
+
+// This file provides a custom `ClaimSpec` type, FuzzClaimSpec, to be used
+// for fuzzing claims within the ClaimPredicate (defined in amber package).
+// FuzzClaimSpec is intended to be used for providing the user with the
+// needed elements to characterize the security of a revision of the source
+// code based on fuzzing.
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/project-oak/transparent-release/pkg/amber"
+)
+
+// FuzzClaimV1 is the URI that should be used as the ClaimType in V1 Amber
+// Claim representing a V1 Fuzz Claim.
+const FuzzClaimV1 = "https://github.com/project-oak/transparent-release/fuzz_claim/v1"
+
+// FuzzClaimSpec gives the `ClaimSpec` definition. It will be included in a
+// Claim, which itself is part of an in-toto statement where the subject
+// refers to a Git repository.
+type FuzzClaimSpec struct {
+	// `ClaimSpec` per fuzz-target.
+	PerTarget []FuzzSpecPerTarget `json:"perTarget"`
+	// `ClaimSpec` for all fuzz-targets.
+	PerProject *FuzzStats `json:"perProject"`
+}
+
+// FuzzSpecPerTarget contains the fuzzing claims specification per fuzz-target.
+type FuzzSpecPerTarget struct {
+	// Name of the fuzz-target.
+	Name string `json:"name"`
+	// Path of the fuzz-target, relative to the root of the Git repository.
+	Path string `json:"path"`
+	// Fuzzing statistics of the fuzz-target.
+	FuzzStatsPerTarget *FuzzStats `json:"fuzzStats"`
```

Comment:
super nit: Feel free to ignore :)

```suggestion
	FuzzStats *FuzzStats `json:"fuzzStats"`
```